### DISCUSSION TITLE
How to put steam friends in workspace 4 and steam and any steam game in workspace 2?

### DISCUSSION THREAD

**Discussion by marc-hg** [dalyIsaac/Whim#863] [Q&A] [answered]
I'd like just the steam friends chat in workspace 4 but steam and any game launched through steam in workspace 2. Is this possible?

**Comment by urob** [comment, marked as answer]
Yes, as long as the different windows differ in *either* title, class or exe, you should be able to create a Routing to do what you want. You can use a tool like `WindowsSpy` to look up these properties for arbitrary windows. Once you identified a match criterion that works, it should be straightforward to adopt the [example in the Wiki](https://dalyisaac.github.io/Whim/docs/customize/routing.html) to fit your needs.

**Comment by marc-hg** [comment]
From what I'm seeing there isn't one for AddProcessMatchRoute right? I wanted to use it for a regex to match all my jetbrains ides to a workspace

**Reply by urob** [reply]
I'd have to check about the general case, but for the Jetbrain IDEs the following should work:
```cs
context.RouterManager.AddWindowClassRoute("SunAwtFrame", "code");  // Pycharm & Rider
```
